packages/ses/test/test-repair-intrinsics.js

```diff
@@ -3,6 +3,8 @@
 
 import test from 'ava';
 import { repairIntrinsics } from '../src/lockdown.js';
+import {getOwnPropertyNames} from '../src/commons.js';
+import {initialGlobalPropertyNames} from '../src/permits.js';
```

Comment:
Run `yarn format` at the top of the repository to clear out the CI errors for these lines.

Reply:
fixed 4cb35cb6e